core/crypto/pedersen_hash.go

```diff
@@ -8,15 +8,15 @@ import (
 
 // PedersenArray implements [Pedersen array hashing].
 //
-// [Pedersen array hashing]: https://docs.starknet.io/documentation/develop/Hashing/hash-functions/#array_hashing
+// [Pedersen array hashing]: https://docs.starknet.io/documentation/architecture_and_concepts/Cryptography/hash-functions/#array_hashing
 func PedersenArray(elems ...*felt.Felt) *felt.Felt {
 	var digest PedersenDigest
 	return digest.Update(elems...).Finish()
 }
 
 // Pedersen implements the [Pedersen hash].
 //
-// [Pedersen hash]: https://docs.starknet.io/documentation/develop/Hashing/hash-functions/#pedersen_hash
+// [Pedersen hash]: https://docs.starknet.io/documentation/architecture_and_concepts/Cryptography/hash-functions/#pedersen_array_hash
```

Comment:
```suggestion
// [Pedersen hash]: https://docs.starknet.io/architecture-and-concepts/cryptography/hash-functions/#pedersen_array_hash
```

Reply:
I think this one should point to `#pedersen_hash` anchor